seccomp_internal.go

```diff
@@ -177,15 +205,15 @@ var (
 	// Error thrown on bad filter context
 	errBadFilter = fmt.Errorf("filter is invalid or uninitialized")
 	// Constants representing library major, minor, and micro versions
-	verMajor = int(C.C_VERSION_MAJOR)
-	verMinor = int(C.C_VERSION_MINOR)
-	verMicro = int(C.C_VERSION_MICRO)
+	verMajor     = uint(C.get_major_version())
```

Comment:
gofmt flags the whitespace on these three lines as incorrect - you can fix with gofmt -s -w